applications/matter_bridge/README.rst

```diff
@@ -112,8 +141,13 @@ User interface
     :end-before: matter_door_lock_sample_led1_end
 
 Button 1:
-     If pressed for six seconds, it initiates the factory reset of the device.
-     Releasing the button within the six-second window cancels the factory reset procedure.
+    Depending on how long you press the button:
+
+    * If pressed for less than three seconds, it initiates the SMP server (Security Manager Protocol).
+      After that, the Direct Firmware Update (DFU) over Bluetooth Low Energy can be started.
+      (See `Upgrading the device firmware`_.)
+    * If pressed for more than three seconds, it initiates the factory reset of the device.
+      Releasing the button within the 3-second window cancels the factory reset procedure.
```

Comment:
Is it really correct calling it canceling the reset if it's starting the SMP server? Or am I misunderstanding the 3-second window?

Reply:
Let me explain it then, it goes like this:
0 s < press time < 3 s - enables DFU over SMP
3 s <= press time < 6 s - initiation of factory reset (basically it's only about printing a warning that factory reset is going to happen soon, so user can react and release the button to cancel it)
press time = 6 s - actual factory reset is done